managers/templates/managers/project_detail.html

```diff
@@ -0,0 +1,96 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% block content %}
+{% load rest_framework %}
+
+<h2>
+  <small>
+    <a href="{% url 'custom-projects-list' %}">
+      <span class="glyphicon glyphicon-chevron-left"></span>
+    </a>
+  </small>
+  &nbsp;{{ project.name }}&nbsp;
+  <small>
+    <a href="#">
+      <span class="glyphicon glyphicon-pencil"></span>
+    </a>
+  </small>
+</h2>
+
+<br/>
+
+<u1>
+  <b>Started at: </b> {{project.start_date}}
+  {% if project.terminated is True %}
```

Comment:
It is enought to do `{% if project.terminated %}`

Reply:
Done